benchmarks/user/benchmark.py

```diff
@@ -98,8 +138,12 @@ def from_value(ctx, param, value):
         click.option('--dle', callback=from_value,
                      type=click.Choice(['noop'] + configuration._accepted['dle']),
                      help='Devito loop engine (DLE) mode'),
-        click.option('-a', '--autotune', is_flag=True,
-                     help='Switch auto tuning on/off')
+        click.option('-bs', '--block-shape', nargs=3, type=(int, int, int),
+                     callback=config_blockshape, multiple=True, is_eager=True,
+                     help='Loop-blocking shape, bypass autotuning'),
+        click.option('-a', '--autotune', default='aggressive', callback=config_autotuning,
```

Comment:
default to basic maybe?

Reply:
mmhh, I think I'm just concerned people trying out devito might forget to set it to advanced. I think I prefer aggressive

Reply:
ok